packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -20,8 +20,7 @@
 				v-else-if="parameter.type === 'notice'"
 				class="parameter-item"
 				:content="$locale.nodeText().inputLabelDisplayName(parameter, path)"
-				:truncate="parameter.typeOptions && parameter.typeOptions.truncate"
-				:truncate-at="parameter.typeOptions && parameter.typeOptions.truncateAt"
+				:truncateAt="parameter.typeOptions && parameter.typeOptions.truncateAt"
```

Comment:
feel free to remove all instances of `truncateAt` and `truncate` as it's not used anywhere